themes/src/main/resources/theme/keycloak.v2/account/src/package-lock.json

```diff
@@ -1,2572 +1,2233 @@
 {
   "name": "keycloak.v2",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
The lockfile version was downgraded because an older version of NPM was used, make sure to use at least NPM v7 to restore this.